CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# Release 1.20.0 14-06-2018
+
+- Add cors_with_regex preset
```

Comment:
can we combine 1, 2, 3 lines into
Add cores_with_regex preset with --cors_allow_origin_regex flag.

remove 2 line

Reply:
done